docs/reference/filters.md

```diff
@@ -1993,3 +1993,57 @@ Example:
 ```
 originMarker("apiUsageMonitoring", "deployment1", "2019-08-30T09:55:51Z")
 ```
+
+## fadeInDuration
+
+When this filter is set, and the route has a load balanced backend, then the newly added endpoints will receive
+the traffic in a gradually increasing way, starting from their detection for the specified duration, after which
+they receive equal amount traffic as the previously existing routes. By default, the fade-in function is linear,
+but it can be adjusted with the [fadeInExponent](#fadeInExponent) filter. The detection time of an load balanced
+backend endpoint is preserved over multiple generations of the route configuration (over route changes). This
+filter can be used to saturate the load of autoscaling applications that require a warm-up time and therefore a
+smooth ramp-up. The fade-in feature can be used together with the round-robin and random LB algorithms.
+
+Parameters:
+
+* the duration of the fade-in in milliseconds or as a duration string
+
+Example:
+
+```
+fadeInDuration("3m")
+```
+
+## fadeInExponent
+
+This filter, when used together with the [fadeInDuration](#fadeInDuration) filter, can be used to adjust the
+shape of the fade-in function, based on the following equation:
+
+current-rate = proportional-rate * (time-since-detection / duration) ^ exponent
```

Comment:
Should it reflect what the rate is when elapsed > duration? 
E.g. like `rate = min((now - detected)/duration, 1) ^ exponent`?

Reply:
Thanks, makes sense!